src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -263,6 +263,10 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.RulesetToEditorconfigConverter" Version="3.0.0" />
```

Comment:
Please this package reference under "build tools" group above and add "PrivateAssets=All". This makes the dependency as "build only", i.e. our runtime package must not depend on it.